ui/raidboss/data/04-sb/alliance/orbonne_monastery.txt

```diff
@@ -301,7 +301,7 @@ hideall "--sync--"
 6004.0 "--sync--" sync /:38C2:Ultima, the High Seraph/ window 1000,0
 6007.0 "Demi-Virgo Line" sync /:Ultima, the High Seraph:38C2:/
 6016.0 "East/West March" sync /:Ultima, the High Seraph:38A[01]:/
-6018.0 "Ray Of Light" sync /:Dominion:38B7:/
+6018.0 "Ray of Light" sync /:Dominion:38B7:/
```

Comment:
This does make me wonder if timeline replacements should also be case insensitive.

Reply:
this would be great